packages/package.json

```diff
@@ -18,8 +18,8 @@
     "lint-fix": "lerna run lint-fix",
     "prettier-fix": "lerna run prettier-fix",
     "test": "lerna run test",
-    "publish": "npm run build && lerna publish",
-    "force-publish": "lerna run tsc && lerna publish --force-publish --no-git-tag-version --no-push"
+    "publish:lerna": "npm run build && lerna publish",
```

Comment:
`publish` is a reserved script name, we shouldn't use it:

https://github.com/lerna/lerna/issues/2824

Reply:
Also related to having those `gitHead` entries:

https://github.com/lerna/lerna/issues/1880

Reply:
renamed `publish` to `release`, now publishing with lerna doesn't display error upon successful package publication.